Rock_paper_scissor_game/index.js

```diff
@@ -0,0 +1,106 @@
+const rock = document.getElementById("rock")
+const paper = document.getElementById("paper")
+const scissor = document.getElementById("scissor")
+const stat = document.getElementById("status")
+stat.innerText = "Start"
+const status_human = document.getElementById("status_human")
+const status_bot = document.getElementById("status_bot")
+
+
+let human_count = 0
+let bot_count = 0
+const human_score = document.getElementById('human')
+human_score.innerText = human_count
+const bot_score = document.getElementById('bot')
+bot_score.innerText = bot_count
+
+const random_number_generator = ()=>(Math.ceil(Math.random()*3))   //random number generator
```

Comment:
use proper spacings.

const random_number_generator = () => (Math.ceil(Math.random()*3))
update the same in the code.